### DISCUSSION TITLE
Use page for submission message

### DISCUSSION THREAD

**Discussion by roland-d** [verbb/formie#1067] [Ideas]
Hi,

Currently I am trying to implement this design:
![image](https://user-images.githubusercontent.com/359377/187622159-a14b2689-e7fe-4c3a-b837-61b2034305c3.png)

What you see here are 3 pages:
1. Fill in fields
2. Summary
3. Submission message

Now this looks nice but it does not work because I have hidden the Submit button on page 3 as this is just the submission text. Formie always checks for the submit on the last page.

My idea is to have an option to show the submission message on a page, much like the `summary` field instead of showing it at the top/bottom/nowhere.

Correct me if I am wrong, currently I would need to restyle the complete tab rendering to get this layout as I see it.

**Comment by engram-design** [comment]
Yeah there's no way to really achieve this through the form builder, as every page is assumed to be part of the submission. That's correct, in order to get this layout, you'd need to use some custom rendering to style the success message as a tab, to that the last tab isn't a page in the form. Certainly doable with custom templates.
